patches/common/chromium/portals_fix_crash_when_detaching_oopif_from_portal.patch

```diff
@@ -0,0 +1,183 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Alexandre=20Lach=C3=A8ze?= <alexandre.lacheze@gmail.com>
+Date: Tue, 25 Jun 2019 17:36:08 +0200
+Subject: Portals: Fix crash when detaching OOPIF from portal.
```

Comment:
I cherry-pick the Chromium commit as it. Not sure it is the expected formatting in the end.